### ISSUE TITLE
Update user data

### ISSUE THREAD

**Issue by Rastopapola** [divio/django-simple-sso#61]
# Status quo
1. Server A, client B, user Bob
2. Bob opens client B from server A for the first time. His data will be used to create a new User in the client's database.
3. Bob leaves the client, goes back to server and changes some user data
4. Bob opens client B from server A again. His data won't be updated in the client's database

# Enhancement

The reason is how the `build_user` method is implemented:
```python


    def build_user(self, user_data):
        try:
            user = User.objects.get(username=user_data['username'])
            # Here should be an update route, which writes all **user_data into the existing user object
        except User.DoesNotExist:
            user = User(**user_data)
        user.set_unusable_password()
        user.save()
        return user
```

**Comment by Rastopapola** [comment]
PullRequest can be found in #62

**Comment by Rastopapola** [comment]
@FinalAngel Hi Angelo! A quick reminder for this issue and the pr #62 . I would love to hear from you guys ;)

**Comment by Rastopapola** [comment]
Has been merged in #62
